README.md

```diff
@@ -74,24 +74,94 @@ jobs:
 ```
 
 ### Uploading Test Results to GitHub
-By default, the `convertReportToXUnit` parameter is set to true. This action generates XML report and converts to xUnit report. You can either upload the XML report as an artifact or publish the results with `Publish Test Results` action which reads the converted xUnit report to review the results on GitHub. See [Publish Test Results](https://github.com/marketplace/actions/publish-test-results) for details.
-
-#### Example
+By default, the `convertReportToXUnit` parameter is set to true. This action generates XML report and converts to xUnit report. You can upload the test reports in the following ways:
+- Upload the XML report to GitHub as an artifact.
+- Publish the results with another action which reads the converted xUnit report to review the results on GitHub. We recommend using [Publish Test Results](https://github.com/marketplace/actions/publish-test-results) and [Test Reporter](https://github.com/marketplace/actions/test-reporter) to publish the results to GitHub.
 
+#### Examples
+#### Upload the reports to GitHub as an artifact
 ```yaml
-# Upload the reports to GitHub as an artifact.
 - name: Upload report artifact
   uses: actions/upload-artifact@v4
   with:
     name: SOAtestReports # Artifact name
     path: /reports # Directory containing files to upload
+```
 
-# Publish the results with `Publish Test Results` action
-- name: Publish Test Results
-  uses: EnricoMi/publish-unit-test-result-action/windows@v2
+#### Publish Test Results
+Prerequisites
```

Comment:
For the prerequisites, I don't think we need to be very detailed. We could simply mention that a Python3 environment is required on the action runner, providing a link to the specific section in the plugin's documentation. 
For permissions, we could include comments in the example YAML and remove the 'Permissions' section. After the example YAML, we should add a description such as: Refer to the [documentation](url) for more information.